package.json

```diff
@@ -24,5 +25,11 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
+  "devDependencies": {
```

Comment:
don't think we need any of these dev dependencies, these are handled by create-react-app.

Reply:
I donot know how it got added but I removed it .